src/react-components/auth-dialog.js

```diff
@@ -22,7 +22,7 @@ class AuthDialog extends Component {
         ) : authOrigin === "spoke" ? (
           <FormattedMessage id="auth.spoke-verified" values={{ br: <br /> }} />
         ) : (
-          <FormattedMessage id="auth.verified" />
+          <FormattedMessage id="auth.verified" values={{ br: <br /> }} />
```

Comment:
interesting -- the approach I've been taking with this is use `white-space: pre` in the CSS. (though I don't recall if I've done it with translation tags.) any opinions on which is better?

Reply:
Ah. Using `white-space: pre` to enforce formatting feels a bit odd to me, but I think you're right, in this particular case it is probably a better solution since we control the white space in the string explicitly. I'll switch these over.